content/articles/accessible-pdf-wheel.md

```diff
@@ -0,0 +1,229 @@
+* [Create an interactive PDF](https://helpx.adobe.com/indesign/how-to/indesign-create-interactive-pdf.html)
+* [Articles panel in InDesign](https://helpx.adobe.com/indesign/how-to/indesign-articles-panel.html)
+
+### 2. Optimize: Adobe Acrobat Pro
+
+The next step was to optimize the interactive PDF worksheet for better
+accessibility using Adobe Acrobat Pro. Tagging and Reading Order tools are the
+building blocks of accessible PDF documents to ensure correct readings by
+screen readers and Adobe PDF Reader’s Read Out Loud functionality.
+
+First, I examined the Identity Wheel worksheet using the Reading Order tool.
+Since I structured my original InDesign document properly, the order of content
+required little fixing. I had to fix some peculiarities unique to InDesign PDF
+export (fractured paragraphs, for example), but other changes were minor.
+
+Next, I repaired some structural tags; for example, improper headings, and list
+items in the worksheet instructions, with the Tagging tool. Finally, to address
+the issue with partial reading of on-screen visible content, I set the the page
+layout to “Single Page” in the “Initial View” document property.
+
+These resources helped me understand Acrobat Pro better for accessible PDF
+optimization:
+
+* [Reading Order and Tagging tool in Acrobat Pro](https://helpx.adobe.com/acrobat/using/touch-reading-order-tool-pdfs.html)
+* [Add Tags to an Untagged Document](https://www.adobe.com/accessibility/products/acrobat/pdf-repair-add-tags.html)
+* [Examine and Repair the Tag Order](https://www.adobe.com/accessibility/products/acrobat/pdf-repair-repair-tags.html)
+* [Examine and Repair the Tag Order—advanced](https://www.adobe.com/accessibility/products/acrobat/pdf-repair-repair-tags-advanced.html)
+
+### 3. Test: Acrobat Pro, Screen reader, Adobe Read Out Loud
+
+Adobe Acrobat Pro comes with an
+[accessibility checker](https://helpx.adobe.com/acrobat/using/create-verify-pdf-accessibility.html#check_accessibility_of_PDFs)
+that provides a report on the level of compliance in a PDF document. I ran the
+Identity Wheel worksheet through the checker after fixing the content reader
+order and tags.
+
+Finally, I tested the worksheet with VoiceOver—the screen reader by Apple—and
+with Acrobat Reader’s Read Out Loud functionality, to make sure that the
+reading and tabbing order was correct, and the accessibility experience
+followed what we had planned.
+
+__Final result: [Accessible interactive Identity Wheel Activity PDF](/img/assets/identity_wheel_compiled.pdf)__
+
+## Lessons learned
+
+What I’ve learned from this whole process are:
+
+1. Accessibility experience __is user experience__ and __is learning
+experience__. Get these right first, and the content will be accessible.
+2. PDF content needs to be in a logical, or meaningful structure and sequential
+order using semantic structure in the original document as much as possible
```

Comment:
`PDF content... before exporting to PDF` doesn't quite gel. This clause is confusing too: `sequential order using semantic structure`. How about: `Content needs to be semantically structured and sequentially ordered before exporting to PDF`?

Reply:
Thanks